tox.ini

```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = security,flake8,black,typing,py
+envlist = pre-commit,pylint,typing,py
```

Comment:
Should we include pre-commit to github actions "checks" workflow? Any other changes needed to the workflow?

Reply:
We already run pre-commit in GitHub (without tox).

Since the pre-commit configuration freezes versions at the pre-commit level, and all other checks use tox, I don’t think we need anything else.